Python/Product/VSInterpreters/Interpreter/LaunchConfigurationUtils.cs

```diff
@@ -25,7 +25,7 @@
 namespace Microsoft.PythonTools.Interpreter {
     public class LaunchConfigurationUtils {
         public static Dictionary<string, string> GetFullEnvironment(LaunchConfiguration config, IServiceProvider serviceProvider, UIThreadBase uiThread) {
-            if (config.Interpreter == null) {
+            if (config.Interpreter == null && config.InterpreterPath == null) {
```

Comment:
Might be better to split this into two separate checks, so that you can throw a different exception using `nameof(InterpreterPath)` for the message.